correlation_study.py

```diff
@@ -0,0 +1,79 @@
+import streamlit as st
+import data_tools as dt
+
+
+def run_correlation_study(raw_docs):
```

Comment:
Good idea to split this out into a separate file and import into `explore.py`  👍